scripts/performance/perf_load/perf_client_fees.py

```diff
@@ -228,3 +236,9 @@ def _on_pool_create_ext_params(self):
                        "pool_fees": self._pool_fees})
         self._logger.info("_on_pool_create_ext_params done {}".format(params))
         return params
+
+
+def random_string(string_length=10):
```

Comment:
I think it's better to re-use/extend helper API from `scripts/performance/perf_load/perf_utils.py`: https://github.com/hyperledger/indy-node/blob/f08c6cf2a64d290bf015dfbb9fd9222da5bdfb1f/scripts/performance/perf_load/perf_utils.py#L54